configs/getConfig.js

```diff
@@ -0,0 +1,20 @@
+const _ = require("lodash");
```

Comment:
I think `config.js` in the top level would probably be fine. 

I'm hesitant to move it into `src` or `bin` as those are different in my mind.

What do you think?